includes/application/forwardApplication.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ *
+ *  to forward application if not processed in certain days.
+ */
+
+session_start();
+
+if ( !( isset( $_SESSION['Username'] ) && isset( $_SESSION['Name'] ) ) )
+{
+	echo "session id :".session_id()." ,You must login first to visit this page.";
+	die();
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->